### DISCUSSION TITLE
最新版的easywechat小程序登录问题

### DISCUSSION THREAD

**Discussion by ryuan1990** [w7corp/easywechat#2645] [Q&A]
$app=new Application($this->config);
提示
Cannot assign EasyWeChat\Kernel\Config to property EasyWeChat\MiniApp\Application::$config of type EasyWeChat\Kernel\Contracts\Config
报错

**Comment by 1459547510** [comment]
检查一下是不是项目里还有一个overtrue/wechat的包，这个包里面使用的是低版本，我之前也遇到了同样的问题

**Reply by phperRaven** [reply]
是的，就是这个问题，感谢

**Reply by JunyuChen-web** [reply]
这个问题具体怎么解决？

**Reply by overtrue** [reply]
> overtrue/wechat
```
composer remove overtrue/wechat
```
